src/elements/Typography.js

```diff
@@ -0,0 +1,16 @@
+import styled from "styled-components";
+import { Box } from "./Box";
+
+import { compose, lineHeight } from "styled-system";
+
+const composedHelpers = compose(lineHeight);
+
+export const Typography = styled(Box)`
+  ${composedHelpers}
+`;
+
+Typography.defaultProps = {
+  // as: 'p',
+  fontWeight: 1
+  // color: 'text500',
```

Comment:
Kullanılmayacaksa yorum tutmak yerine doğrudan kaldırılabilir.

Reply:
Kullanma konusunda karasız olduğum için yorum satırına ekledim